stable/kong/README.md

```diff
@@ -72,6 +73,7 @@ and their default values.
 | proxy.nodePort                    | Node port when service type is `NodePort`                              |                       |
 | proxy.type                        | k8s service type. Options: NodePort, ClusterIP, LoadBalancer           | `NodePort`            |
 | proxy.loadBalancerIP              | To reuse an existing ingress static IP for the admin service           |                       |
+| proxy.loadBalancerSourceRanges    | Limit admin access to CIDRs if set and service type is `LoadBalancer`  | `[]`                  |
```

Comment:
The description needs update here, it seems like a copy paste error.

Reply:
Done!